config/sidekiq_scheduler.yml

```diff
@@ -99,3 +99,8 @@ VBADocuments::UploadRemover:
   cron: "15,45 * * * * * America/New_York"
   class: VBADocuments::UploadRemover
   description: "Clean up submitted documents from S3"
+
+HCASubmissionFailureEmailAnalytics:
```

Comment:
shouldn't this be `HCA::SubmissionFailureEmailAnalyticsJob`?

Reply:
Sure, since that's more expressive.  I looked through the `sidekiq_scheduler.yml` file and there doesn't seem to be a hard standard on naming things.  Some include the subclass, some don't.  Some include 'Job', some don't.   In general, I agree.  More explicit is better.

Reply:
`HCA::SubmissionFailureEmailAnalyticsJob` will need to put either here or in the `class` attribute, i think it's a good idea to just name the job the same thing as the class and omit the class attribute